core/math/projection.cpp

```diff
@@ -313,6 +314,48 @@ void Projection::set_perspective(real_t p_fovy_degrees, real_t p_aspect, real_t
 	*this = *this * cm;
 }
 
+void Projection::set_oblique(real_t p_fovy_degrees, real_t p_aspect, real_t p_z_near, real_t p_z_far, Quaternion p_oblique_plane, bool p_flip_fov) {
+	if (p_flip_fov) {
+		p_fovy_degrees = get_fovy(p_fovy_degrees, 1.0 / p_aspect);
+	}
```

Comment:
I don't understand what p_flip_fov is for

Reply:
this isn't part of my PR, This is just part of the Projection::set_perspective code. Almost everything about the Oblique camera is a duplication of the Perspective camera because it was the most straightforward way to implement oblique near plane clipping for my project when I originally started. This is why I've made the suggestion to switch to a more general approach using a boolean toggle to enable oblique near plane clipping for any camera projection type.